From cda70ff6e9204163ff69178abb865257a75f3972 Mon Sep 17 00:00:00 2001
From: Mauro Rossi <issor.oruam@gmail.com>
Date: Wed, 17 Oct 2018 09:38:05 +0200
Subject: [PATCH 3/3] intel: annotate the intel genx helpers as public

intel_is_genx() and intel_get_genx() which are now private
were actually being used by drm_gralloc_intel,
this patch annotates them as drm_public
---
 intel/intel_chipset.c | 4 ++--
 intel/intel_chipset.h | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/intel/intel_chipset.c b/intel/intel_chipset.c
index f6e37ee7..b36d7f04 100644
--- a/intel/intel_chipset.c
+++ b/intel/intel_chipset.c
@@ -46,7 +46,7 @@ static const struct pci_device {
 	INTEL_SKL_IDS(9),
 };
 
-drm_private bool intel_is_genx(unsigned int devid, int gen)
+drm_public bool intel_is_genx(unsigned int devid, int gen)
 {
 	const struct pci_device *p,
 		  *pend = pciids + sizeof(pciids) / sizeof(pciids[0]);
@@ -68,7 +68,7 @@ drm_private bool intel_is_genx(unsigned int devid, int gen)
 	return false;
 }
 
-drm_private bool intel_get_genx(unsigned int devid, int *gen)
+drm_public bool intel_get_genx(unsigned int devid, int *gen)
 {
 	const struct pci_device *p,
 		  *pend = pciids + sizeof(pciids) / sizeof(pciids[0]);
diff --git a/intel/intel_chipset.h b/intel/intel_chipset.h
index 0a48e0da..4fa0a07d 100644
--- a/intel/intel_chipset.h
+++ b/intel/intel_chipset.h
@@ -331,8 +331,8 @@
 #include <stdbool.h>
 #include <libdrm_macros.h>
 
-drm_private bool intel_is_genx(unsigned int devid, int gen);
-drm_private bool intel_get_genx(unsigned int devid, int *gen);
+drm_public bool intel_is_genx(unsigned int devid, int gen);
+drm_public bool intel_get_genx(unsigned int devid, int *gen);
 
 #define IS_GEN9(devid) intel_is_genx(devid, 9)
 #define IS_GEN10(devid) intel_is_genx(devid, 10)
-- 
2.30.2

